projects/ngneat/forms-manager/src/lib/forms-manager.ts

```diff
@@ -462,4 +619,24 @@ export class NgFormsManager<FormsState = any> {
   private removeInitialValue(name: FormKeys<FormsState>) {
     coerceArray(name).forEach(name => this.initialValues$$.delete(name));
   }
+
+  private markDescendantsAsDirty(
+    control: AbstractControl,
+    options?: {
+      onlySelf?: boolean;
+      emitEvent?: boolean;
+    }
+  ) {
+    control.markAsDirty(options);
+
+    if (control instanceof FormGroup || control instanceof FormArray) {
+      Object.values(control.controls).forEach((control: AbstractControl) => {
```

Comment:
Object.values requires a polyfill for unsupported browsers.

Reply:
I use now [`Object.keys`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) instead of `Object.values`. Is this okay for you?